### ISSUE TITLE
new installers with checksum not compatible with arm64

### ISSUE THREAD

**Issue by mickkael** [argoproj/argo-cd#6143]
If you are trying to resolve an environment-specific issue or have a one-off question about the edge case that does not require a feature then please consider asking a question in argocd slack [channel](https://argoproj.github.io/community/join-slack).

Checklist:

* [x] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [x] I've included steps to reproduce the bug.
* [x] I've pasted the output of `argocd version`.

**Describe the bug**

When building argocd with rpi4, it breaks during the installer, as the checksum files for arm64 are not included

**To Reproduce**

Build with arm64 architecture

**Expected behavior**

build without issue

**Version**

latest master code

**Logs**

```
build-argocd-kt74x-3206545311: + grep -q helm-v2.17.0-linux-arm64.tar.gz /tmp/installers/checksums/helm-v2.17.0-linux-arm64.tar.g
z.sha256
build-argocd-kt74x-3206545311: grep: /tmp/installers/checksums/helm-v2.17.0-linux-arm64.tar.gz.sha256: No such file or directory
build-argocd-kt74x-3206545311: + echo No checksum for helm-v2.17.0-linux-arm64.tar.gz in /tmp/installers/checksums/helm-v2.17.0-l
inux-arm64.tar.gz.sha256
build-argocd-kt74x-3206545311: No checksum for helm-v2.17.0-linux-arm64.tar.gz in /tmp/installers/checksums/helm-v2.17.0-linux-ar
m64.tar.gz.sha256
build-argocd-kt74x-3206545311: + exit 1
build-argocd-kt74x-3206545311: v1.21.9error building image: error building stage: failed to execute command: waiting for process
to exit: exit status 1
```

**Comment by vielmetti** [comment]
Looking forward to seeing this!

**Comment by mickkael** [comment]
Fixed with #6192
